pkg/sdk/testint/tables_integration_test.go

```diff
@@ -0,0 +1,949 @@
+		require.NoError(t, err)
+
+		require.Equal(t, 2, len(tableDetails))
+		assert.Empty(t, tableDetails[0].PolicyName)
+	})
+
+	t.Run("alter table: set and unset tags", func(t *testing.T) {
+		name := random.String()
+		id := sdk.NewSchemaObjectIdentifier(database.Name, schema.Name, name)
+		columns := []sdk.TableColumnRequest{
+			*sdk.NewTableColumnRequest("COLUMN_1", sdk.DataTypeVARCHAR),
+			*sdk.NewTableColumnRequest("COLUMN_2", sdk.DataTypeVARCHAR),
+		}
+
+		err := client.Tables.Create(ctx, sdk.NewCreateTableRequest(id, columns))
+		require.NoError(t, err)
+		t.Cleanup(cleanupTableProvider(id))
+
+		columnTags := []sdk.TagAssociationRequest{
+			{
+				Name:  sdk.NewSchemaObjectIdentifier(database.Name, schema.Name, tag1.Name),
+				Value: "v1",
+			},
+			{
+				Name:  sdk.NewSchemaObjectIdentifier(database.Name, schema.Name, tag2.Name),
+				Value: "v2",
+			},
+		}
+
+		alterRequest := sdk.NewAlterTableRequest(id).WithSetTags(columnTags)
+		err = client.Tables.Alter(ctx, alterRequest)
+		require.NoError(t, err)
+
+		returnedTagValue, err := client.SystemFunctions.GetTag(ctx, tag1.ID(), id, sdk.ObjectTypeTable)
+		require.NoError(t, err)
+
+		assert.Equal(t, "v1", returnedTagValue)
+
+		returnedTagValue, err = client.SystemFunctions.GetTag(ctx, tag2.ID(), id, sdk.ObjectTypeTable)
+		require.NoError(t, err)
+
+		assert.Equal(t, "v2", returnedTagValue)
+
+		unsetTags := []sdk.ObjectIdentifier{
+			tag1.ID(),
+			tag2.ID(),
+		}
+		alterRequestUnsetTags := sdk.NewAlterTableRequest(id).WithUnsetTags(unsetTags)
+
+		err = client.Tables.Alter(ctx, alterRequestUnsetTags)
+		require.NoError(t, err)
+
+		_, err = client.SystemFunctions.GetTag(ctx, tag1.ID(), id, sdk.ObjectTypeTable)
+		require.Error(t, err)
+
+		_, err = client.SystemFunctions.GetTag(ctx, tag2.ID(), id, sdk.ObjectTypeTable)
+		require.Error(t, err)
+	})
+
+	t.Run("alter table: drop columns", func(t *testing.T) {
+		name := random.String()
+		id := sdk.NewSchemaObjectIdentifier(database.Name, schema.Name, name)
+		columns := []sdk.TableColumnRequest{
+			*sdk.NewTableColumnRequest("COLUMN_1", sdk.DataTypeVARCHAR),
+			*sdk.NewTableColumnRequest("COLUMN_2", sdk.DataTypeVARCHAR),
+		}
+
+		err := client.Tables.Create(ctx, sdk.NewCreateTableRequest(id, columns))
+		require.NoError(t, err)
+		t.Cleanup(cleanupTableProvider(id))
+
+		alterRequest := sdk.NewAlterTableRequest(id).
+			WithColumnAction(sdk.NewTableColumnActionRequest().WithDropColumns([]string{"COLUMN_1"}))
+		err = client.Tables.Alter(ctx, alterRequest)
+		require.NoError(t, err)
+
+		table, err := client.Tables.ShowByID(ctx, id)
+		require.NoError(t, err)
+
+		currentColumns := getTableColumnsFor(t, client, table.ID())
+		expectedColumns := []expectedColumn{
+			{"COLUMN_2", sdk.DataTypeVARCHAR},
+		}
+		assertColumns(t, expectedColumns, currentColumns)
+
+		assert.Equal(t, table.Comment, "")
+	})
+
+	// TODO: check added constraints
```

Comment:
next pr, leave it for the future, or forgot to implement in this pr ?

Reply:
Pr description:
```
Operations NOT supported in this PR:
- check constraints (...)
```